src/shared/file_op.c

```diff
@@ -2710,6 +2710,14 @@ FILE * wfopen(const char * pathname, const char * mode) {
     FILE * fp;
     int i;
 
+    if (pathname[0] == '\\' || pathname[0] == '/') {
```

Comment:
Could you check if the pointer is valid before performing a comparison?

Reply:
Done!